Wikipedia/Categories/NSString+Extras.m

```diff
@@ -103,6 +103,14 @@ - (NSString*)wmf_stringByCapitalizingFirstCharacter {
     return self;
 }
 
+- (NSString*)wmf_stringByCapitalizingAllCharactersIfDeviceLocaleIsEnglish {
```

Comment:
Since you do this for all languages now, `…IfDeviceLocaleIsEnglish` is a bit of a lie now.
Instead of renaming, at this point, If it is just a single line of code, you don't really wrap it in a method anymore.